test/test_files/abl_sampling/abl_sampling.inp

```diff
@@ -22,7 +22,7 @@ time.checkpoint_interval      =  5       # Steps between checkpoint files
 incflo.gravity          =   0.  0. -9.81  # Gravitational force (3D)
 incflo.density             = 1.0          # Reference density
 
-io.derived_outputs = mag_vorticity "components(velocity,0,1)" "grad(temperature)"
+io.derived_outputs = mag_vorticity "components(gp,0,1)" "grad(temperature)"
```

Comment:
we should(?) be able to leave velocity here

Reply:
yep, that's not a problem. I changed it here so that I would have something different available for the sampling derived outputs

Reply:
you can have them be different if you want.

Reply:
oh, I missed that. I'll revert this part of the PR.

Reply:
```suggestion
io.derived_outputs = mag_vorticity "components(velocity,0,1)" "grad(temperature)"
```